.github/workflows/readme_oc_path_and_rpc.yml

```diff
@@ -0,0 +1,36 @@
+name: README OpenConfig Path and RPC Coverage
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+  schedule:
+    - cron: "49 0 * * *"
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: stable
+        cache: false
+
+    - name: Validate Template README
+      run: |
+        go run tools/validate_readme_spec/validate_readme_spec.go -alsologtostderr doc/test-requirements-template.md; rm -rf tmp
+
+    - name: Validate Bad Example
+      run: |
+        if go run tools/validate_readme_spec/validate_readme_spec.go -alsologtostderr tools/validate_readme_spec/invalid.md; then
+          echo "Validation passed, but failure expected"
+          exit 1
+        fi
+        rm -rf tmp
+
+    - name: Validate Test READMEs
+      run: |
+        # TODO: Add READMEs here that have converted to the new format.
```

Comment:
With a syntax change in it's yaml section, this readme could be added: 

https://github.com/openconfig/featureprofiles/blob/main/feature/bgp/policybase/otg_tests/import_export_multi/README.md

Reply:
In a future PR, we could change the default to parse README with changes perhaps?  Or introduce a flag as an exception in metadata.textproto for all old README's that haven't had a yaml stanza added.

Reply:
I will make the change in a follow-up PR to validate the file.

I propose to parse all READMEs all the time, since that's the only way to know the repo is in good state all the time.

I think the best way to identify READMEs is to parse all `README.md` files under `/features` repo, which is what `addrundata` also does. `addrundata` also validates that whenever a `_test.go` file exists, `README.md` must also exist, which helps our check as well.

In fact, I propose only to tolerate `ErrNotFound` any README that does not have an accompanying `_test.go` file. This way we won't need allow-lists either.

Reply:
I would like to cover README yaml path validation even when the README is standalone / no _test.go file.

Reply:
I'm still proposing to validate all README.md under /features **where the heading exists**, no matter whether a _test.go file is present. However, I'm proposing that if a _test.go file doesn't exist, we don't force the issue. The benefit of this is that we don't need an allow-list.

WDYT? -- an allow-list or don't force the test author to write such a section if a _test.go file doesn't exist?

Reply:
I feel like we should require the section to exist without exceptions.    (It requires converting existing README before merging these checks in).  Worst case an author can add an empty section with a TODO comment?

Reply:
Alright, allow-list approach implemented here: https://github.com/openconfig/featureprofiles/pull/2881

Reply:
thanks!